packages/adaptive-ui-designer-figma-plugin/src/figma/node.ts

```diff
@@ -541,6 +548,40 @@ export class FigmaPluginNode extends PluginNode {
         }
     }
 
+    /**
+     * Cleans up stroke values based on what we're about to apply.
+     * 
+     * @remarks
+     * Figma has a default invisible stroke width of `1`. If you add a stroke in the UI
+     * it sets the weight to `1` and adds a default color.
+     * If you change the weight, then remove the strokes (nothing visible) it maintains
+     * the old stroke weight, but when you add again it resets to `1`.
+     *
+     * @param node - The Figma node
+     * @param values - The entire list of values to be applied
+     */
+    private handleStroke(values: AppliedStyleValues) {
+        const applyingFill = values.has(StyleProperty.borderFillTop)
+            || values.has(StyleProperty.borderFillRight)
+            || values.has(StyleProperty.borderFillBottom)
+            || values.has(StyleProperty.borderFillLeft);
+        const applyingThickness = values.has(StyleProperty.borderThicknessTop)
+            || values.has(StyleProperty.borderThicknessRight)
+            || values.has(StyleProperty.borderThicknessBottom)
+            || values.has(StyleProperty.borderThicknessLeft);
+
+        if (applyingFill && applyingThickness) {
+            if (isContainerNode(this._node) || isIndividualStrokesShapeNode(this._node)) {
+                (this._node as IndividualStrokesMixin).strokeTopWeight = 0;
+                (this._node as IndividualStrokesMixin).strokeRightWeight = 0;
+                (this._node as IndividualStrokesMixin).strokeBottomWeight = 0;
+                (this._node as IndividualStrokesMixin).strokeLeftWeight = 0;
+            } else {
+                // If it's not "individual" we're going to set it anyway.
```

Comment:
I'm not certain what this means since it's not actually doing anything. Is the `else` necessary?

Reply:
The comment was to address why the condition was there at all. I've moved and rewritten it.